Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -243,20 +249,88 @@ In calculating the SCI value, you SHOULD use the highest granularity data availa
+| [Software Carbon Intensity](#reporting-the-sci-value) - reason for not selecting `R` baseline from provided items| sci/r-source-comment | SHALL | Text | This item SHALL be provided, only where the `sci/r-source` element does not contain one of the identifiers in the [pre-set list](#preset-list-for-baselines). |
+| Product Name, entity name, or Software Product | metadata/name | MUST | Text | |
+| Contact name | metadata/contact-name | MUST | Text | The point of contact responsible and accountable for the report. |
+| Contact email | metadata/contact-email | MUST | Text | The point of contact responsible and accountable for the report. |
+| Organization | metadata/organization | SHOULD | Text | This SHALL be populated where the report is not by an individual contributor, otherwise it is not required. |
+| GUID | metadata/guid | MUST | GUID | Following a format in [RFC4122], provided to uniquely identify this particular product, resource, or service. |
+| Version | metadata/version | MAY | Text | |
+| Date of Calculation | metadata/calculation-date | MUST | Date | Following a format described in [RFC3339], [a subset](https://ijmacd.github.io/rfc3339-iso8601/) of ISO 8601 |
+| Further information on calculation | metadata/calculation-information | MAY | Text | More information on your calculation methodology can be provided as freetext, or as a URL to an external document or software repository. |
+| Further information on report | metadata/report-information-url | MAY | URL | More information on your calculation methodology may be provided, and this SHALL be a URL to an external document or software repository.  |
+
+
+<!--
+
+- Supporting infrastructure and systems contained within the [Software Boundary](#software-boundary)
+-
+  - `CI` MUST be reported
+  - `C` SHOULD be reported, but where it is not reported, a reason MUST be provided
+  - The `R` baseline used MUST be reported
+  - If the `R` baseline used wa not selected from the  for baselines, then a reason for that choice MUST be provided
+- Metadata for the SCI value being reported
+  - Product name MUST be reported
+  - Contact details, including a name and email, for a point of contact responsible for the report MUST be provided
+  - Organization SHOULD be provided, where the report is not by an individual contributor
+  - A GUID, following a format described in [RFC4122], MUST be provided to uniquely identify this particular product, resource, or service.
+  - A software version for the software product about which this report is made, MAY be provided.
+  - The date the calculation was made, following a format described in [RFC3339], MUST be provided.
+  - More information on your calculation methodology MAY be provided, as a freetext comment, limited to 300 characters.
+  - More information on your report MAY be provided, as a URL linking to an external document or software repository.
+
```

Comment:
Oh yes, I think this shouldn't be a comment, it should be the main thrust of this section!!! 

But also I would suggest separate it out into two headings to make it easier to get an understanding via a glance.

- mandatory reporting requirements
- optional reporting requirements


Also is there any technical need to limit to 300 chars? That's mighty small, a tweet or two.

Reply:
I was actually going to say it's the opposite way round, the table is harder to read for me and since the rest of the doc is in this language so makes sense for this section to be in this language as well, more human readable and parseable.

We should have an appendix which goes into specific detail about how exactly to report and this table does a great job of that, it works very well together with the json/yaml examples.

Reply:
Perfect - I'll move the table to the appendix, and keep the free text in the main body of the spec. 

Promise I'm not from https://www.reddit.com/r/totallynotrobots/